viewer/packages/cad-geometry-loaders/src/sector/SectorDownloadScheduler.ts

```diff
@@ -58,10 +71,43 @@ export class SectorDownloadScheduler {
     });
   }
 
+  private createDiscardedConsumedSector(sector: WantedSector): ConsumedSector {
+    return {
+      modelIdentifier: sector.modelIdentifier,
+      metadata: sector.metadata,
+      levelOfDetail: LevelOfDetail.Discarded,
+      group: undefined,
+      instancedMeshes: undefined
+    };
+  }
+
+  private abortLoadOfDiscardedSector(sector: WantedSector, sectorIdentifier: string): Promise<ConsumedSector> | null {
+    // Abort pending downloads
+    const pendingSector = this._pendingSectorDownloads.get(sectorIdentifier);
+    if (pendingSector !== undefined) {
+      pendingSector.abort();
+      return pendingSector.consumedSector;
+    }
```

Comment:
Remove comment and refactor to separate function/method. Call it something like abortPendingDownloads.